cpp/include/cudf/lists/list_device_view.cuh

```diff
@@ -93,7 +102,10 @@ class list_device_view {
   }
 
   /**
-   * @brief Checks whether element is null at specified index in the list row.
+   * @brief Checks whether the element is null at the specified index in the list row.
```

Comment:
We should be consistent about whether or not we refer to rows or not in this documentation. I think we generally don't want to because once the list_device_view is constructed the abstraction is independent of the original lists column from which we're viewing a row.
```suggestion
   * @brief Checks whether the element is null at the specified index in the list
```